src/components/Organisers.js

```diff
@@ -49,7 +49,7 @@ const Organisers = () => (
                 <div className="overlay">
                   <div className="organisers-item-meta">
                     <h5>Mathew Finch</h5>
-                    <p>Head of Emerging Technology at nib health funds</p>
+                    <p>Head of Emerging Tech, nib health funds</p>
```

Comment:
We should make these link to twitter or something...

Reply:
I'll add a link when you click the button